### ISSUE TITLE
Error link HREF_BASE gives 404 errors

### ISSUE THREAD

**Issue by richiejarvis** [ably/ably-java#750]
Hi Team,

In line 187 of https://github.com/ably/ably-java/blob/main/lib/src/main/java/io/ably/lib/types/ErrorInfo.java, the link is given as https://help.ably.io/error/, but this gives a 404 error.

Is there a new page which gives a list of all errors and an explanation?  
Can the code be corrected to point to the correct link please?

Cheers,

Richie

**Comment by QuintinWillison** [comment]
Ah, good spot @richiejarvis. That'll be something that didn't move over when we went from `ably.io` to `ably.com`. Does [errorsHelp.json](https://github.com/ably/ably-common/blob/main/protocol/errorsHelp.json) help? Otherwise, perhaps reach out to Website team or Mark in your team.

**Comment by richiejarvis** [comment]
Yes it helps!  Just reporting the error in the ErrorInfo.java class
